stream-chat-android-ui-components/src/main/res/values/styles.xml

```diff
@@ -229,6 +229,9 @@
         <item name="streamUiThreadAvatarStyle">@style/StreamUi.Thread.Avatar</item>
         <item name="streamUiReplyAvatarStyle">@style/StreamUi.Reply.Avatar</item>
         <item name="streamUiMessageOptionsAvatarStyle">@style/StreamUi.MessageOptions.Avatar</item>
+        <item name="streamUiAvatarInitialsTextColor">@color/stream_ui_white</item>
+        <item name="streamUiAvatarInitialsTextSize">20sp</item>
+        <item name="streamUiAvatarColor">@color/stream_ui_black</item>
```

Comment:
I was wondering if it would be better to provide the same default values when we initialize the custom attributes on `AvatarStyles.` For example, we're initializing with `stream_ui_accent_blue` color for the `AvatarColor` on `AvatarStyle`. And same for the `streamUiGlobalAvatarBorderColor`, `streamUiAvatarInitialsTextSize`, and `streamUiAvatarBorderWidth`.

Reply:
I aligned the initializations, but `streamUiAvatarBorderWidth` has many different values. I think we should leave as is so we avoid uncessary changes in the API.